docs/pages/web/toast.js

```diff
@@ -156,36 +158,13 @@ If  confirmation toast's text with more complex style is required, such as bold
 </Flex>
 `}
           />
-        </MainSection.Subsection>
-        <MainSection.Subsection title="Error">
           <MainSection.Card
-            cardSize="lg"
-            defaultCode={`
-<Flex justifyContent="center" width="100%">
-  <Toast
-    text="You're offline."
-    variant="error"
-    primaryAction={{ accessibilityLabel: 'Reconnect', label: 'Reconnect' }}
-  />
-</Flex>
-`}
-          />
-        </MainSection.Subsection>
-
-        <MainSection.Subsection title="Thumbnail">
-          <MainSection.Card
-            cardSize="lg"
+            cardSize="md"
             defaultCode={`
-<Flex justifyContent="center" width="100%">
+<Flex direction="column" justifyContent="center" width="100%" gap={4}>
   <Toast
-    text={
-      <Text inline>
-        Saved to{' '}
-        <Link display="inline" target="blank" href="https://www.pinterest.com/search/pins/?q=home%20decor">
-          Home decor
-        </Link>
-      </Text>
-    }
+    text="Your account admin rights were successfully saved"
+    primaryAction={{ accessibilityLabel: 'Undo', label: 'Undo' }}
     thumbnail={
```

Comment:
This needs to change, yes? You may want to move the current Toast examples out to separate files to catch errors like this one.

Reply:
DONE